File: Blaze/CollectionViews/BlazeTilesCollectionView.m
//
//  BlazeTilesCollectionView.m
//  Blaze
//
//  Created by Bob de Graaf on 07-10-15.
//  Copyright © 2015 GraafICT. All rights reserved.
//

#import "BlazeTilesCollectionView.h"
#import "BlazeTileCollectionViewCell.h"

@implementation BlazeTilesCollectionView

-(instancetype)initWithCoder:(NSCoder *)coder
{
    self = [super initWithCoder:coder];
    if(!self) {
        return nil;
    }
    
    //Register Cells
    [self registerNib:[UINib nibWithNibName:XIBDefaultTileCollectionViewCell bundle:[NSBundle bundleForClass:NSClassFromString(XIBDefaultTileCollectionViewCell)]] forCellWithReuseIdentifier:XIBDefaultTileCollectionViewCell];
    
    return self;
}

@end

File: Blaze-Swift/CollectionViews/BlazeTilesCollectionView.swift
//
//  BlazeTilesCollectionView.swift
//  Blaze-Swift
//
//  Created by Sebastiaan Seegers on 10-11-17.
//  Copyright © 2017 ZoinksInteractive. All rights reserved.
//

import UIKit

class BlazeTilesCollectionView: UICollectionView {

    var ID: Int = 0
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        let bundle: Bundle = Bundle(for: NSClassFromString("BlazeTileCollectionViewCell")!)
        let nib: UINib = UINib(nibName: "BlazeTileCollectionViewCell", bundle: bundle)
        self.register(nib, forCellWithReuseIdentifier: "BlazeTileCollectionViewCell")
    }
}
